src/kernel/include/shared/cos_types.h

```diff
@@ -187,6 +187,9 @@ typedef enum {
 
 #define CAPTBL_EXPAND_SZ 128
 
+#define COS_VIRT_MACH_COUNT        2
```

Comment:
I wasn't sure, VM_COUNT sounded a bit ambiguous, Virtual Memory or Virtual Machine, so I was being explicit in this case. But when I put all these in vkernel, I'll name them VM. I guess, over there, I would have to be explicit about Memory not Machine. 

I'll put them in vkernel component, have a header within that and do it. Perhaps even the capabilities offsets.